samples/test-samples.cmake

```diff
@@ -42,10 +42,12 @@ endif ()
 if (BUILD_ENCLAVES)
   set(SAMPLES_LIST debugmalloc file-encryptor helloworld log_callback
                    switchless)
+  set(CRYPTO_LIB_LIST mbedtls mbedtls mbedtls mbedtls mbedtls)
   # Debug malloc will set allocated memory to a fixed pattern.
   # Hence do not enable pluggable_allocator test under USE_DEBUG_MALLOC.
   if (COMPILER_SUPPORTS_SNMALLOC AND NOT USE_DEBUG_MALLOC)
     list(APPEND SAMPLES_LIST pluggable_allocator)
+    list(APPEND CRYPTO_LIB_LIST mbedtls)
```

Comment:
What is the relation between snmalloc and mbedtls?

Reply:
This means that the sample is linking against oecryptombedtls instead of oecryptoopenssl.

Reply:
Basically, each sample in the list is paired with either mbedtls or openssl, which indicates with or crypoto library the sample will use. This provides the flexibility if later we want to test some samples with OpenSSL.